### PR TITLE
Added tests for  hammer organization subcommands

### PR DESCRIPTION
**Pull request by svtkachenko**
Added coverage for hammer `organization` subcommands. Also covers #4082.

```python
% pytest -v tests/foreman/cli/test_organization.py -k 'parameter or location'
===================================================================== test session starts =====================================================================
platform linux2 -- Python 2.7.12, pytest-3.0.4, py-1.4.31, pluggy-0.4.0 -- /home/qui/code/venv/2/bin/python2
cachedir: .cache
rootdir: /home/qui/code/robottelo, inifile: 
plugins: xdist-1.15.0, html-1.12.0, cov-2.4.0
collected 67 items 

tests/foreman/cli/test_organization.py::OrganizationTestCase::test_positive_add_location_by_id <- robottelo/decorators/__init__.py PASSED
tests/foreman/cli/test_organization.py::OrganizationTestCase::test_positive_add_location_by_name <- robottelo/decorators/__init__.py PASSED
tests/foreman/cli/test_organization.py::OrganizationTestCase::test_positive_add_parameter <- robottelo/decorators/__init__.py PASSED
tests/foreman/cli/test_organization.py::OrganizationTestCase::test_positive_remove_location_by_id <- robottelo/decorators/__init__.py SKIPPED
tests/foreman/cli/test_organization.py::OrganizationTestCase::test_positive_remove_location_by_name <- robottelo/decorators/__init__.py SKIPPED
tests/foreman/cli/test_organization.py::OrganizationTestCase::test_positive_remove_parameter <- robottelo/decorators/__init__.py SKIPPED
tests/foreman/cli/test_organization.py::OrganizationTestCase::test_positive_update_parameter <- robottelo/decorators/__init__.py PASSED

===================================================================== 60 tests deselected =====================================================================
===================================================== 4 passed, 3 skipped, 60 deselected in 79.54 seconds =====================================================
```

### CODE REVIEW

**Comment by svtkachenko** [comment]
Also added in separate commit alphabetical sorting of subcommand helpers according to its parameter name, i.e. `add_domain` -> `remove_domain` -> `add_environment` -> ...
Just a bit of eye candy that eases navigation.

File: tests/foreman/cli/test_organization.py [hunk 1]
```diff
@@ -1162,6 +1164,176 @@ def test_positive_remove_capsule_by_name(self):
...
+        Org.add_location({
+            'location': loc['name'],
+            'name': org['name'],
+        })
+        result = Org.info({'id': org['id']})
+        self.assertEqual(len(result['locations']), 1)
+        self.assertIn(loc['name'], result['locations'])
+
+    @run_only_on('sat')
+    @skip_if_bug_open('bugzilla', 1395229)
```
Comment (oshtaier) [inline tests/foreman/cli/test_organization.py]: What is a connection between that bug and that test case?

Comment (oshtaier) [inline tests/foreman/cli/test_organization.py]: Today, I created a bug that seems much more relevant:
https://bugzilla.redhat.com/show_bug.cgi?id=1408940

Comment (svtkachenko) [inline tests/foreman/cli/test_organization.py]: @oshtaier When enabling debug mode you can see that hammer sends API requests and remove-* commands send '[]' as value.

Comment (svtkachenko) [inline tests/foreman/cli/test_organization.py]: I'm afraid that bug would closed with the same resolution as this one: https://bugzilla.redhat.com/show_bug.cgi?id=1379944

Comment (oshtaier) [inline tests/foreman/cli/test_organization.py]: Hmm, description has nothing in common with problem. Can we update it in some way?

File: tests/foreman/cli/test_organization.py [hunk 2]
```diff
@@ -1162,6 +1164,176 @@ def test_positive_remove_capsule_by_name(self):
...
+        """
+        param_name = gen_string('alpha')
+        org = make_org()
+        Org.set_parameter({
+            'name': param_name,
+            'value': gen_string('alpha'),
+            'organization': org['name'],
+        })
+        result = Org.info({'id': org['id']})
+        self.assertEqual(len(result['parameters']), 0)
```
Comment (pondrejk) [inline tests/foreman/cli/test_organization.py]: Hi @svtkachenko, I know this is currently skipped but I don't see where you remove the parameter in this test. Is there something missing?

Comment (svtkachenko) [inline tests/foreman/cli/test_organization.py]: Indeed! Thank you for noticing :)

**Comment by pondrejk** [comment]
ACK

**Comment by rochacbruno** [comment]
@svtkachenko this PR has conflicts to be solved/rebased

**Comment by svtkachenko** [comment]
@rochacbruno Rebased.

**Pull request merged**